lib/langchain/vectorsearch/qdrant.rb

```diff
@@ -64,6 +64,16 @@ def update_texts(texts:, ids:)
       add_texts(texts: texts, ids: ids)
     end
 
+    # Remove a list of texts from the index
+    # @param ids [Array<Integer>] The ids to remove
+    # @return [Hash] The response from the server
+    def remove_texts(ids:)
```

Comment:
Hmm... `remove_texts()` or should we call it `delete_texts()`?

Reply:
I think remove is better because it is the opposite of "add".

However we can change it with delete. What do you prefer?